syntax = "proto3";
package agent;




import "model/agent_admin/agent.proto";

/*
ListAgent请求
*/
message ListAgentRequest {
    /*
    agentUUID集合
    */
    repeated string agentIds = 1 ;
}

/*
ListAgent返回
*/
message ListAgentResponse {
    /*
    页数
    */
    int32 page = 1 ;
    /*
    页大小
    */
    int32 page_size = 2 ;
    /*
    总数
    */
    int32 total = 3 ;
    /*
    数据列表
    */
    repeated agent_admin.Agent list = 4 ;
}

/*
ListAgentApi返回
*/
message ListAgentResponseWrapper {
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    返回码解释
    */
    string codeExplain = 2 ;
    /*
    错误详情
    */
    string error = 3 ;
    /*
    返回数据
    */
    agent.ListAgentResponse data = 4 ;
}

